src/wrapped_radix_engine_toolkit/low_level/notarized_transaction.rs

```diff
@@ -2,63 +2,106 @@ use crate::prelude::*;
 
 #[derive(Debug, Clone, Eq, PartialEq, uniffi::Record)]
 pub struct NotarizedTransaction {
-    pub signed_intent: SignedIntent,
+    signed_intent: SignedIntent,
```

Comment:
Hide constructor so that we ensure we always go via `new` which now is a failable constructor, performing `compile` - checking SBOR depth.